src/content/en/fundamentals/instant-and-offline/get-started/graphicalcontent-4.md

```diff
@@ -0,0 +1,186 @@
+
+### Browser-side Resizing
+
+Often, large images are retrieved from the server and then resized in the browser with CSS. This might be for a perfectly logical reason -- such as using just one image for a thumbnail that expands on hover -- or it might just be careless coding. Whatever the reason, it wastes bandwidth.
+
+For example, you might have a 1200 x 600 pixel image that you present at 60 x 30 (as a 5% thumbnail) and roll it up to full size on hover using a CSS transition. It works fine and looks great, but if the user never actually hovers over the thumbnail, then literally 95% of the time that image took to download was wasted. You may be better off taking the time to make separate, properly sized, thumbnails and use them in those locations, displaying the full sized images only if they're actually requested.
+
+Of course, using separate images invokes a tradeoff in additional HTTP requests. It is worth testing to see which technique -- reducing the number of images or reducing the number of HTTP requests -- saves the most load time. You might also consider combining graphical resources as a balanced approach, as described in the next article, **HTTP Requests**.
+
+But thumbnails aren't the only images that need physical resizing. All your images should be appropriately sized for their intended use and should not rely on the browser to resize them for rendering.
+
+### Physical Resizing
+
+Once you've determined the size that an image should really be, how do you get it to that size but still retain the quality and characteristics it needs? The first thing you need is a good image manipulation tool, of which there are many. Here are a few to consider.
+
+#### ResizeIt
+
+[https://itunes.apple.com/us/app/resizeit/id416280139](https://itunes.apple.com/us/app/resizeit/id416280139?mt=12)
+
+This Mac-only desktop product lets you change the size of multiple images simultaneously, and can convert file formats at the same time. 
+
+![ResizeIt](images/image_400.png)
+
+#### PicResize
+
+[http://www.picresize.com/](http://www.picresize.com/)
+
+One of several good browser-based tools that gives you lots of options for cropping, rotating, resizing, adding effects to, and converting images.
+
+![PicResize](images/image_401.png)
+
+#### Gimp
+
+[https://www.gimp.org/](https://www.gimp.org/)
+
+This ever-popular cross-platform tool just gets better with age. Powerful and flexible, Gimp lets you perform a wide variety of image manipulation tasks including, of course, resizing.
+
+![Gimp](images/image_402.jpg)
+
+This is just a smattering of available tools. We aren't here for a comparative review of image editors, although CrazyEgg has a comprehensive article here: 
+
+[https://www.crazyegg.com/blog/image-editing-tools/](https://www.crazyegg.com/blog/image-editing-tools/)
+
+The point is, there are other facets to improving image speed beyond raw size.
+
+## Image File Considerations
```

Comment:
General comment for this section: We need to be careful when talking about image quality as a lower filesize when you perceive an image to be about the same quality doesn't always mean that one format or optimizer is better than another. I'd recommend reading https://kornel.ski/en/faircomparison as this highlights a reasonable way to approach quality/filesize.